fili-core/src/main/java/com/yahoo/bard/webservice/data/PartialDataHandler.java

```diff
@@ -39,24 +87,17 @@
      * present for a given combination of request metrics and dimensions (pulled from the API request and generated
      * druid query) at the specified granularity.
      *
-     * @param constraint  Constraint containing all the column names the request depends on
-     * @param physicalTables  the tables whose column availabilities are checked
+     * @param availableIntervals  the intervals available in the tables
      * @param requestedIntervals  The intervals that may not be fully satisfied
      * @param granularity  The granularity at which to find missing intervals
      *
      * @return subintervals of the requested intervals with incomplete data
      */
     public SimplifiedIntervalList findMissingTimeGrainIntervals(
```

Comment:
it's the most general part of the contract.  I'm inclined to leave it visible, and we certainly can't make it less visible without a deprecation period.